### PR TITLE
Enforcing Automatic Code formatting

### PR DESCRIPTION
**Pull request by schuylermartin45**
This PR contains no human changes, outside of edits to the `Makefile`.

I ran `black` and `isort` against the entire project, in preparation for a future linting fix PR.

From this point forward, `black` and `isort` will run against all files in Percy.

`black` and `isort` have been configured to play nice with each other AND to work without fighting against the `pylintrc` file we've adapted from Google's linting rules.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -103,8 +103,8 @@ lint:			## runs the linter against the project
 	pylint --rcfile=.pylintrc $(LINTER_FILES)
 
 format:			## runs the code auto-formatter
-	isort --profile black --line-length=80 $(LINTER_FILES) percy/tests/
-	black --line-length=80 $(LINTER_FILES) percy/tests/
```
Comment (JeanChristopheMorinPerso) [inline Makefile:107]: You should probably remove the `LINTER_FILES` variable from the Makefile too.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (schuylermartin45) [reply Makefile:107]: It's still being used for the actual linter, which has not been fully enabled yet. Linting only occurs on a few files.

Mark and I talked yesterday and decided to get the formatter in first and slowly enable the linter over time.

**Review by cbouss — APPROVED** [review]
LGTM, I'd wish for more than 80 line length though.

File: Makefile [hunk 2]
```diff
@@ -103,8 +103,8 @@ lint:			## runs the linter against the project
...
+	isort --profile black --line-length=80 percy
```
[the remark this answers is not in the document]
Comment (JeanChristopheMorinPerso) [reply Makefile:106]: Oh indeed. It's pretty common to see something between 110 and 120.

**Comment by schuylermartin45** [comment]
`black` defaults to 88, Google's linting rules enforce 80. There was a method to the madness :)

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (markan) [reply Makefile:106]: I kinda lean towards a longer length as well. While I remember the VT100, I don't miss it :-)

**Comment by schuylermartin45** [comment]
By popular demand: https://github.com/anaconda-distribution/percy/pull/53

**Pull request merged**